platform-ui-skin/src/main/webapp/skin/less/_global.less

```diff
@@ -1138,4 +1138,8 @@ hr {
     direction:rtl;
     max-width: 100%;
     margin: auto;
+}
+
+#UISiteBody {
+    padding: 20px;
```

Comment:
pay attention to this because it will be applied on all sites including intranet and customers existing sites.

Reply:
Done!